native/native.tests/testData/programName/main.c

```diff
@@ -0,0 +1,14 @@
+#include <stdio.h>
+#include <unistd.h>
+
+int main(int argc, char *argv[]) {
+    printf("calling exec...\n");
+    fflush(NULL);
+
+    // Kotlin executable name is in argv[1]
+    // Forward argv[2..n] to kotlin executable as arguments (the first one should be the programName according to posix)
+    execv(argv[1], &(argv[2]));
+
+    printf("exec failed\n");
+    return 0;
```

Comment:
Let's return non-zero exit code?

Reply:
Good point. I also now print the errno for easier debugging in case the `exec` syscall fails.